credits/Citing-OSCAR.md

```diff
@@ -23,8 +23,8 @@ If you are using **BibTeX**, you can use the following BibTeX entries:
   key          = {OSCAR},
   organization = {The OSCAR Team},
   title        = {OSCAR -- Open Source Computer Algebra Research system,
-                  Version 1.0.0},
-  year         = {2024},
+                  Version {{ site.data.oscar_version.latest_release }}},
+  year         = {2025},
```

Comment:
Likewise here, replace 2025 by the current year?

Reply:
No, but it should be taken from the YAML.

This is also how we do it for the GAP website, see https://github.com/gap-system/GapWWW/blob/master/_data/release.json

Reply:
Just updated.